src/adserver.js

```diff
@@ -0,0 +1,57 @@
+import {formatQS} from './url';
+
+//Adserver parent class
+var adserver = function(attr) {
+  var that = {};
+  that.name = attr.adserver;
+  that.code = attr.code;
+  that.getWinningBidByCode = function() {
+    var bidObject = $$PREBID_GLOBAL$$._bidsReceived.find(bid => bid.adUnitCode === that.code);
+    return bidObject;
+  };
+  return that;
```

Comment:
If you call this function as a constructor with the `new` keyword, you can remove lines 5 and 12 as these will then happen implicitly. Then change the 4 other `that`s to `this` and capitalize `adserver` on line 4 to prevent jshint from giving a warning.